core/Module.md

```diff
@@ -0,0 +1,84 @@
+# Module kotlinx-io-core
+
+The module provides core multiplatform IO primitives and integrates it with platform-specific APIs.
+
+`kotlinx-io` aims to provide a concise but powerful API along with efficient implementation.
```

Comment:
Maybe something like `kotlinx-io core` is better, because this is a file with a description of `kotlinx-io-core`, and here the text is written as if about the whole IO?